src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -26,6 +26,8 @@
       url: /accessibility/alternative-text-for-images
     - title: Describing a button’s purpose
       url: /accessibility/button-purpose
+    - title: Announcements
```

Comment:
The order of the nav items here is looking really strange. **Accessibility at GitHub** and **Guidelines** should in theory be the first links. The rest of the links should probably be revisited but I'm not sure what order, and [Describing a button’s purpose](https://primer.style/design/accessibility/button-purpose) could fit more with the other docs titles - @ichelsea was working on a PR for this but not sure what's its status.

Reply:
I re-ordered the nav items in https://github.com/primer/design/pull/237/commits/35fb348f9aa32749d7299f35c4c46e3616568b61, loosely following the order of WCAG Success Criteria:

| Page Title | Related WCAG |
|:---:|:---:|
| Accessibility at GitHub | - |
| Guidelines | - |
| Semantic HTML | 1.3.1 |
| Headings | 1.3.2 |
| Alternative text for images | 1.1.1 |
| Text resize and respacing | 1.4.4 |
| Assistive technology announcements | - |
| Focus management | 2.4.3 |
| Describing a button’s purpose | 2.4.4 |
| Tools | - |

Reply:
@yaili 
> was working on a PR for this but not sure what's its status.

Yeah! I wanted to wait till my links doc and headings update were merged before I messed with it because I tried to do all my editing in one go and that messed up my links doc PR 😅

Reply:
@smockle I would put "Tools" underneath "Guidance". 

I can delete my PR if you're going to solve it here.

Reply:
@ichelsea “Tools” has been moved in https://github.com/primer/design/pull/237/commits/264211ecd47fa0fa20e44eccfe70b507a8452703 👍 

Re: deleting other PRs—I was following @yaili’s recommendation in https://github.com/primer/design/pull/237#discussion_r831576564:
> …It would be good if we could reorder as part of this PR…

I recall a bigger discussion somewhere about adding subheadings to the nav 🤔

Reply:
Yeah! No worries. I can whittle down this PR: https://github.com/primer/design/pull/227 to just add in the accessibility menu page if we do the reordering in this one. 

The subheadings sound like a larger issue tied into a hope for primer.style redesign I think